_posts/2016-xx-xx-release-0.16.md

```diff
@@ -119,6 +119,116 @@ If you are using the low-level qcprot algorithm your self intead of our provided
 wrappers you have to change your code since the API has changed. For more see
 the [CHANGELOG].
 
+## MemoryReader: Reading trajectories from memory
+
+MDAnalysis typically reads trajectories from files on-demand, so that it can efficiently deal with large trajectories - even those that do not fit in memory. However, in some cases, both for convenience and for efficiency, it can be an advantage to work with trajectories directly in memory. In this release, we have introduced a MemoryReader, which makes this possible.
+
+The MemoryReader works with numpy arrays, using the same format as that used by for instance `DCDReader.timeseries()`. You can create a Universe directly from such an array:
+
+```python
+from MDAnalysis import Universe
+from MDAnalysisTests.datafiles import DCD, PDB_small
+from MDAnalysis.coordinates.memory import MemoryReader
+
+# Create a Universe using a DCD reader
+universe = Universe(PDB_small, DCD)
+
+# Extract coordinates
+coordinates = universe.trajectory.timeseries()
```

Comment:
@kain88-de I wanted to illustrate how easy it is to generate a Universe from an existing numpy array, which is actually one of the prime use-cases when we use it in our group. Also, it is actually quite convenient that MemoryReader supports the same interface as `.timeseries`, including the `format` option for specifying which dimension is which. It is therefore something I would really like to highlight. If you have strong opinions about this, I can certainly remove it, but otherwise, I would prefer to keep it in. Perhaps we can add a `transfer_to_memory` example first, and then this afterwards?